rwkv/chat_with_bot.py

```diff
@@ -16,102 +16,43 @@
 
 # English, Chinese
 LANGUAGE: str = 'English'
-
-# True: Q&A prompt
-# False: chat prompt (you need a large model for adequate quality, 7B+)
-QA_PROMPT: bool = False
+# 1: Q&A prompt 
+# 2: chat prompt (you need a large model for adequate quality, 7B+)
+QA_PROMPT: int = 2 
+
+PROMPT_FILE: str = f'./rwkv/prompt/default/{LANGUAGE}-{QA_PROMPT}.py'
```

Comment:
Since prompt files contain only text, I strongly suggest to use JSON for them (or any other widely supported text/config format), because executing arbitrary Python file looks a little scary.

For example, a prompt file can look like this:
```
prompts/english-ai-assistant.json
{
    "separator": ":",
    "bot": "Bot",
    "user": "User",
    "prompt": "\nThe following is a verbose..."
}
```
There is no need to format the JSON manually, the files can be generated using `json.dump`.

---

Other suggestions are for the file structure:
- if there is no specific need for having `default` subdirectory, I think it should be removed
- instead of using numbers like `1`, `2`, `4`, it would be more readable to use short descriptions like `qna`, `ai-assistant`, `girl`